src/Components/About/About.module.css

```diff
@@ -0,0 +1,50 @@
+.heading h3 {
+  font-weight: 500;
+  font-size: 36px;
+  line-height: 60px;
+  color: #373535;
+  margin-bottom: 30px;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
```

Comment:
On your containers you should specify a max-width so that at a certain point your UI stops growing with the width of the screen. On large monitors it can be harder to perceive.